rules/linux/persistence_linux_backdoor_user_creation.toml

```
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2023/03/07"
+integration = ["system"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/04/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the attempt to create a new backdoor user by setting the user's UID to 0. Attackers may alter a user's UID to 
+0 to escalate privileges or establish persistence on a system or domain.
```

```suggestion
0 to escalate privileges or establish persistence on a system.
```
Just a suggestion, idk if this works on samba domains or something like that